app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,57 @@
+class ReviewsController < ApplicationController
+  def new
+    @bourbon = Bourbon.find(params[:bourbon_id])
+    @user = User.find(current_user.id)
```

Comment:
You should be able to use `@user = current_user` instead of this line throughout this controller.